CONTRIBUTING.md

```diff
@@ -190,15 +190,46 @@ of their patch is not influenced by their setup or implicit knowledge of the sys
 
 ## Code Style
 
-We use automated formatting with Fourmolu to enforce a unified style across the code bases. It is checked in the CI process.
-After installing Fourmolu 0.12, there are some makefile targets to help formatting
-the code base.
-
+We use automated formatting, checked in the CI process, to enforce a unified
+style across the code bases and include these makefile targets to help with
+formatting:
 
 * `make style` - Format the `Cabal`, `Cabal-syntax` and `cabal-install` directories.
 * `make style-modified` - Format files modified in the current tree.
 * `make style-commit COMMIT=<ref>` - Format files modified between HEAD and the given reference.
 
+> [!NOTE]
+> We use `fourmolu-0.12.0.0` for formatting. If installing it with `cabal
+> install`, please make sure to use a version of `GHC >= 9.2.1 && < 9.8`.
+> Its requirement of `GHC2021` sets the lower bound, and its reliance on
+> `ghc-lib-parser` sets the upper bound on GHC versions. A command for
+> installing it this way is:
+>
+> ```
+> $ cabal install fourmolu-0.12.0.0 --overwrite-policy=always --ignore-project
+> ```
+
+> [!Tip]
+> If you have multiple versions of GHC installed with `ghcup`, a series specific
+> ghc can be used by adding an option of `--with-compiler=ghc-x.y` (for the
+> latest installed `x.y.z` version in the `x.y` series) or an option of
+> `--with-compiler=ghc-x.y.z` to the above install command.
+>
+> ```
+> $ ghc --version | sed 's/[[:alpha:]|(|,|[:space:]]//g'
+> 9.10.1
+>
+> $ ghc-9.6 --version | sed 's/[[:alpha:]|(|,|[:space:]]//g'
+> 9.6.6
+>
+> $ ghc-9.4.8 --version | sed 's/[[:alpha:]|(|,|[:space:]]//g'
+> 9.4.8
```

Comment:
Isn't this equivalent to `ghc --numeric-version`?

Reply:
It is, in versions that have it… but I'm pretty sure we don't support ghc 7.0 or earlier any more.

Reply:
Thanks for the tip @jasagredo. I'm using `--numeric-version` now.